Hymns/Common/TabView/TabBar.swift

```diff
@@ -6,39 +6,73 @@ import SwiftUI
  */
 struct TabBar<TabItemType: TabItem>: View {
 
+    @Environment(\.sizeCategory) var sizeCategory: ContentSizeCategory
     @Binding var currentTab: TabItemType
     let tabItems: [TabItemType]
 
     var body: some View {
-        ScrollView(.horizontal, showsIndicators: false) {
-            HStack {
-                ForEach(tabItems) { tabItem in
-                    Spacer()
-                    Button(
-                        action: {
-                            withAnimation(.default) {
-                                self.currentTab = tabItem
-                            }
-                    },
-                        label: {
-                            Group {
-                                if self.isSelected(tabItem) {
-                                    tabItem.selectedLabel
-                                } else {
-                                    tabItem.unselectedLabel
+        Group {
+            if sizeCategory.isAccessibilityCategory() {
+                ScrollView(.horizontal, showsIndicators: false) {
+                    HStack {
+                        ForEach(tabItems) { tabItem in
+                            Spacer()
+                            Button(
+                                action: {
+                                    withAnimation(.default) {
+                                        self.currentTab = tabItem
+                                    }
+                            },
+                                label: {
+                                    Group {
+                                        if self.isSelected(tabItem) {
+                                            tabItem.selectedLabel
+                                        } else {
+                                            tabItem.unselectedLabel
+                                        }
+                                    }.accessibility(label: tabItem.a11yLabel).padding()
+                            })
+                                .accentColor(self.isSelected(tabItem) ? .accentColor : .primary)
+                                .anchorPreference(
+                                    key: FirstNonNilPreferenceKey<Anchor<CGRect>>.self,
+                                    value: .bounds,
+                                    transform: { anchor in self.isSelected(tabItem) ? .some(anchor) : nil }
+                            )
+                            Spacer()
+                        }
+                    }
+                }
+            } else {
+                HStack {
```

Comment:
I don't think this works, because now this is what it looks like without the scroll view:
![Simulator Screen Shot - iPhone 8 - 2020-06-29 at 11 49 29](https://user-images.githubusercontent.com/1427524/86044254-b48a0f80-b9fe-11ea-9c75-facdd945ae22.png)